pkgs/servers/tailscale/default.nix

```diff
@@ -26,6 +27,15 @@ buildGoModule {
   };
   vendorHash = "sha256-pYeHqYd2cCOVQlD1r2lh//KC+732H0lj1fPDBr+W8qA=";
 
+  patches = [
+    # Reverts "cmd/tailscaled/tailscaled.service: revert recent hardening"
+    (fetchpatch {
+      url = "https://github.com/tailscale/tailscale/commit/2889fabaefc50040507ead652d6d2b212f476c2b.patch";
```

Comment:
As per the commit message, it says it broke `Debain Stretch`, did it cause any impact to NixOS ?